doc/reference/guidelines.rst

```diff
@@ -263,25 +321,380 @@ Inside these 3 groups, the imported lines are alphabetically sorted.
     from openerp.addons.web.controllers.main import login_redirect
 
 
-Idioms
-------
+Idiomatics Python Programming
+-----------------------------
+
+- Each python file should have ``# -*- coding: utf-8 -*-`` as first line.
+- Always favor *readability* over *conciseness* or using the language features or idioms.
+- Don't use ``.clone()``
+
+.. code-block:: python
+
+    # bad
+    new_dict = my_dict.clone()
+    new_list = old_list.clone()
```

Comment:
Of course that's bad as nor `[].clone()` nor `{}.clone()` actually exist :)
I think you mean `{}.copy()` but `[].copy()` does not exist neither. Not sure why you'd say `{}.copy()` is bad though.